Beecrowd.Beginner.Solutions/1036-BashkarasFormula.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Beecrowd
+{
+    class BhaskarasFormula
+    {
+        public static void Run()
+        {
+            string[] inputs = Console.ReadLine().Split(' ');
```

Comment:
Use var everywhere instead of types.

Reply:
I don't agree with this discipline :D

Reply:
Is there any logical thought behind it? It's better to follow the same convention in the project. probably you can get some ideas in this link: [When to Use and Not Use Variable Type var in C#](https://intellitect.com/blog/when-to-use-and-not-use-var-in-c/)